styles/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+    text-align: center;
+    background-image: url(../assets/birds-cute-wallpaper-pattern-brightened.jpg);
+    margin: 0;
+}
+
+header, footer {
+    border: 2px solid rgb(188, 188, 188);
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    background-color: #eab031;
+    color: rgb(72, 72, 72);
+}
+
+h1 {
+    font-size: max(2vw, 20px);
+}
+
+nav ul {
+    padding: 0;
+}
+
+nav ul li {
+    display: inline;
+    padding: 0.5vw;
+}
+
+a {
+    outline: none;
+    text-decoration: none;
+    font-size: max(1vw, 10px);
+    color: rgb(72, 72, 72);
+}
+
+a:hover { color: rgb(116, 116, 116); }
+
+footer > p {
+    font-size: max(1vw, 10px);
+}
```

Comment:
Really nice CSS overall, y'all did a great job with selectors and ensuring that rulesets aren't repeated between individual files.